act-rules-format.bs

```diff
@@ -235,158 +257,198 @@ When all expectations are true for a test target, the test target `passed` the r
----------------------------------
-
-When representing the test target in the output data, it is often impractical or impossible to serialize the test target as a whole. Instead of this, a pointer can be used to indicate where the test target exists within the web page or other [test subject](#output-test-subject). There are a variety of pointer methods available, such as those defined in [Pointer Methods in RDF 1.0](https://www.w3.org/TR/Pointers-in-RDF/) [[Pointers-in-RDF]].
-
-The pointer method used in the output data of an ACT Rule MUST include the pointer method used in [Test cases](#test-cases).
-
-
-Outcome {#output-outcome}
--------------------------
-
-The definition of a rule MUST always result in one of the following outcomes:
-
-- **Passed**: All [expectations](#expectations) for the [Test Target](#output-test-target) were true
-- **Failed**: One or more expectations for the Test Target was false
-- **Inapplicable**: There were no Test Targets in the [Test Subject](#output-test-subject)
+<div class=example>
+  <p>Outcome of this rule map to the accessibility requirements in the following manner:</p>
+  <ul>
+    <li>`failed`: The success criteria are *not satisfied*</li>
+    <li>`passed` or `inapplicable`: Further testing is necessary to determine if the success criteria are satisfied</li>
+  </ul>
+</div>
 
-<div class="note">
-  While *inapplicable* is a valid result for ACT Rules, it may not be a valid result for all [accessibility requirements](#structure-accessibility-requirements). Notably the success criteria of WCAG 2.0 and WCAG 2.1 can only be evaluated to true (passed) or false (failed). This translation of results is part of [rule aggregation](#rule-aggregation)
+<div class=note>
+  In WCAG 2.1, success criteria do not evaluate to `passed`, `failed` or `inapplicable`. Rather they can be *satisfied* (or not). (See the [WCAG 2.1 definition: satisfies a success criterion](https://www.w3.org/TR/WCAG21/#dfn-satisfies).) If a success criterion is *not satisfied* a web page can only conform if there is a conforming alternative version, as described in [WCAG 2.1 Conformance Requirement 1](https://www.w3.org/TR/WCAG21/#cc1).
 </div>
 
-<div class="note">
-  In addition to `Passed` `Failed` and `Inapplicable`, [[EARL10-Schema]] also defined an `Incomplete` outcome. While this should never be the outcome of a rule when applied in its entirety, it often happens that rules are only partially executed. For example, when applicability was automated, but the expectations have to be evaluated manually. Such "interim" results can be expressed with the "Incomplete" outcome.
+<div class=note>
+  In WCAG 2.1, all content must satisfy the success criteria. Because of this, all ACT Rules for WCAG 2.1 will require a `passed` outcome for every test target, in order for the test subject to be `passed`. While this logic of "every element must pass for the page to pass" will be common, it is not required by the ACT Rules Format.
```

Comment:
I don't understand what this means but I think it is already sufficiently addressed by my proposed rewrite, and by the example above.